src/main/resources/templates/fragments/members.html

```diff
@@ -29,28 +29,28 @@ <h1 class="font-weight-bold text-dark pt-2 mb-0 d-inline-block">All Members
         </div>
 
         <div class="table-responsive-sm">
-            <table class="table table-striped table-hover" aria-live="assertive">
+            <table class="table table-striped table-hover" aria-atomic="true">
                 <thead>
                 <tr>
-                    <th scope="col" ng-click="sortBy('groupingMembers', 'pagedItemsMembers', 'name')" role="button">
+                    <th id="nameColumn" scope="col" ng-click="sortBy('groupingMembers', 'pagedItemsMembers', 'name')" aria-sort="none">
```

Comment:
Could you use kebab-case for all of the IDs added in this PR? So nameColumn should be name-column. Using kebab-case is the best practice, and it also applies to classes.